docker-compose.server.yml

```diff
@@ -10,7 +10,7 @@ networks:
 
 services:
   phpfpm:
-    image: itkdev/php7.4-fpm:alpine
+    image: itkdev/php8.1-fpm:alpine
```

Comment:
Why not PHP 8.3?

Reply:
Not changed yet. Need to ensure everything is working, before updating.